server/main.js

```diff
@@ -152,8 +151,8 @@ process.on('unhandledRejection', (reason, p) => {
 
 function getRequestConfig(method, body){
     const config = {
-        method,
-        body,
+        method: method,
```

Comment:
```
method: method,
body: body
```
can be:
```
method,
body
```

Reply:
Its the same. [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#New_notations_in_ECMAScript_2015](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#New_notations_in_ECMAScript_2015)